### PR TITLE
ES-853397-Add fallback font code snippets in UG documentation

### PR DESCRIPTION
**Pull request by LokeshBaskar**
Hi All,

Task ID: https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/159323

With this MR, we have documented the fallback font functionality for DocIO and Presentation UG. And also modified the API of the Refresh in chart.

Regards,
Lokesh B

### CODE REVIEW

File: Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34310.174
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fallback-fonts-for-custom-range", "Fallback-fonts-for-custom-range\Fallback-fonts-for-custom-range.csproj", "{5339BECC-6DDE-48E9-9A7A-C77C7E47A70B}"
```
Comment (Ramaraj-Marimuthu) [inline Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range.sln:6]: @LokeshBaskar - Please change the sample name to `"Fallback-fonts-for-Unicode-range"`

File: Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range/Program.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Opens the file as stream.
+            using FileStream inputStream = new FileStream(@"../../../Template.docx", FileMode.Open, FileAccess.Read);
+            //Loads an existing Word document file stream.
+            using WordDocument wordDocument = new WordDocument(inputStream, Syncfusion.DocIO.FormatType.Docx);
+            //Adds fallback font for "Arabic" custom unicode range.
```
Comment (Ramaraj-Marimuthu) [inline Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range/Program.cs:15]: @LokeshBaskar - Please modify the `custom` to `specific`

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range/Program.cs (hunk 2 above)
Comment (LokeshBaskar) [reply Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range/Program.cs]: Modify the custom to specific

File: Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range.sln (hunk 1 above)
Comment (LokeshBaskar) [reply Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range.sln]: Changed the sample name to "Fallback-fonts-for-Unicode-range"

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**